### PR TITLE
Click elements with Javascript again.

### PR DESCRIPTION
**Pull request by muddasheep**
With different ways of handling input (radio/checkbox) and buttons (submit) (thanks to PV).

### CODE REVIEW

File: lib/WWW/WebKit2/Locator.pm [hunk 1]
```diff
@@ -288,19 +322,10 @@ sub is_visible {
...
-    $search .= "
-        function isVisible(e) {
-            if (e == undefined) {
-                return 0;
-            }
-
-            if(getComputedStyle(e).visibility === 'hidden') {
-                return 0;
-            }
+    my $is_visible_function = $self->is_visible_function;
```
Comment (jscarty) [inline lib/WWW/WebKit2/Locator.pm:329]: The pod for this should be updated

**Review by jscarty — CHANGES_REQUESTED** [review]
All good, the pod for is_visible needs to be updated. There is no problem with the code it's just a pod update.

File: lib/WWW/WebKit2/Locator.pm (hunk 1 above)
Comment (muddasheep) [reply lib/WWW/WebKit2/Locator.pm:329]: done!

**Review by jscarty — COMMENTED** [review]
ACK!

**Review by jscarty — APPROVED** [review]

**Pull request merged**